docs/userGuide.md

```diff
@@ -174,81 +147,74 @@ Its important to remark that this component's tests are End To End tests, that h
 This requirements are the following:
 * An instance of MongoDB running in `localhost`.
 * An instance of Orion Context Broker running in the location configured in `testConfig.js` (defaults to the alias `oriondb`).
-This instance has to have the 1026 and 27017 open for connections coming from the Grunt tester.
+This instance has to have the 1026 and 27017 open for connections coming from the tests.
+
+### Project build
+The project is managed using npm.
+
+For a list of available task, type
 
-### Libraries
-[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Chai](http://chaijs.com/) Assertion Library + [Sinon](http://sinonjs.org/) Spies, stubs.
+```bash
+npm run
+```
+
+The following sections show the available options in detail.
+
+### Testing
+[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Should.js](https://shouldjs.github.io/) Assertion Library.
 
-The test environment is preconfigured to run [BDD](http://chaijs.com/api/bdd/) testing style with
-`chai.expect` and `chai.should()` available globally while executing tests, as well as the [Sinon-Chai](http://chaijs.com/plugins/sinon-chai) plugin.
+The test environment is preconfigured to run BDD testing style.
 
 Module mocking during testing can be done with [proxyquire](https://github.com/thlorenz/proxyquire)
 
-### Execution
 To run tests, type
-```bash
-grunt test
-```
 
-Tests reports can be used together with Jenkins to monitor project quality metrics by means of TAP or XUnit plugins.
-To generate TAP report in `report/test/unit_tests.tap`, type
 ```bash
-grunt test-report
```

Comment:
There are yet a 3 references (at least) to grunt in this file. Please check lines 57, 62 and 76.

Reply:
Fixed with 41edfcdf6b4da736051e4894ce69f5622376da9a